src/FinalMileCarrier/FinalMileCarrier.php

```diff
@@ -0,0 +1,154 @@
+    private const POSTE_ITALIANE = 'Poste Italiane';
+    private const JAPAN_POST = 'Japan Post';
+    private const KOREA_POST = 'Korea Post';
+    private const LATVIJAS_PASTS = 'Latvijas Pasts';
+    private const LIETUVOS_PAŠTAS = 'Lietuvos paštas';
+    private const POSTLUXEMBOURG = 'POSTLuxembourg';
+    private const POS_MALAYSIA = 'Pos Malaysia';
+    private const MALTA_POST = 'Malta Post';
+    private const MEXICO = 'Correos De Mexico';
+    private const POSTNL = 'PostNL';
+    private const NEW_ZEALAND_POST = 'New Zealand Post';
+    private const NIGERIAN_POSTAL_SERVICE = 'Nigerian Postal Service';
+    private const POSTEN_NORGE = 'Posten Norge';
+    private const PHLPOST = 'PHLPost';
+    private const POCZTA_POLSKA = 'Poczta Polska';
+    private const CTT = 'CTT';
+    private const POȘTA_ROMÂNĂ = 'Poșta Română';
+    private const RUSSIA_POST = 'Russia Post';
+    private const SINGAPORE_POST = 'Singapore Post';
+    private const SLOVENSKÁ_POŠTA = 'Slovenská pošta';
+    private const POŠTA_SLOVENIJE = 'Pošta Slovenije';
+    private const SOUTH_AFRICAN_POST_OFFICE = 'South African Post Office';
+    private const CORREOS = 'Correos';
+    private const POSTNORD = 'PostNord';
+    private const SWISS_POST = 'Swiss Post';
+    private const CHUNGHWA_POST = 'Chunghwa Post';
+    private const THAILAND_POST = 'Thailand Post';
+    private const PTT = 'PTT';
+    private const ROYAL_MAIL = 'Royal Mail';
+    private const CORREO_URUGUAYO = 'Correo Uruguayo';
+    private const USPS = 'USPS';
+
+    private const FINAL_MILE_CARRIERS = [
+        'AR' => ['name' => self::CORREO_ARGENTINO, 'url' => 'https://www.correoargentino.com.ar/formularios/oidn'],
+        'AU' => ['name' => self::AUSTRALIA_POST, 'url' => 'https://auspost.com.au/mypost/track/#/details/%s'],
+        'AT' => ['name' => self::ÖSTERREICHISCHE_POST, 'url' => 'https://www.post.at/sv/sendungsdetails?snr=%s'],
+        'BE' => ['name' => self::BPOST, 'url' => 'https://track.bpost.cloud/btr/web/#/search?lang=fr&itemCode=%s&postalCode=3080'],
+        'BR' => ['name' => self::CORREIOS_DE_BRASIL, 'url' => 'https://rastreamento.correios.com.br/app/index.php'],
+        'BG' => ['name' => self::BULGARIAN_POSTS, 'url' => 'https://www.bgpost.bg/IPSWebTracking/IPSWeb_item_events.asp?itemid=%s&submit=Follow'],
+        'CA' => ['name' => self::CANADA_POST, 'url' => 'https://www.canadapost-postescanada.ca/track-reperage/en#/details/%s'],
+        'CL' => ['name' => self::CORREOS_CHILE, 'url' => 'https://www.correos.cl/web/guest/seguimiento-en-linea?codigos=%s#0'],
+        'HR' => ['name' => self::HRVATSKA_POŠTA, 'url' => 'https://posiljka.posta.hr/Tracking/Details?Barcode=%s'],
+        'CY' => ['name' => self::CYPRUS_POST, 'url' => 'https://ips.cypruspost.gov.cy/ipswebtrack/IPSWeb_item_events.aspx?itemid=%s&Submit=Submit'],
+        'CZ' => ['name' => self::ČESKÁ_POŠTA, 'url' => 'https://www.postaonline.cz/en/trackandtrace/-/zasilka/cislo?parcelNumbers=%s'],
```

Comment:
Are we sure that the special characters will agree with PHP though? I am in favor of replacing them with more limited charset.